src/generator/AutoRest.Ruby/Templates/ServiceClientTemplate.cshtml

```diff
@@ -79,6 +79,62 @@ module @Settings.Namespace
     end
 
     @EmptyLine
+    #
+    # Makes a request and returns the body of the response
```

Comment:
minor: In general for consistency i'd keep each comment line ending with `.` as we have for initialize method as well.